wikimedia/wikipedia-ios
# Native Editor - Add link and image support

**Phabricator:** 
https://phabricator.wikimedia.org/T348071

### Notes
Implements syntax highlighting and actions for link and images.

### Test Steps
1. Visit article.
2. Confirm links and labelled links are syntax highlighted.
3. Confirm image links are syntax highlighted (will look something like `[[File:...]]`)
4. Tap inside of link. Confirm link buttons show selected state and image button shows disabled state.
5. Tap selected link button. Confirm link edit wizard appears. Change link and confirm changes reflect in textView.
6. Tap outside of a link. Confirm image button shows as enabled.
7. Tap image button. Go through wizard to insert an image. Confirm image wikitext inserts into textView.
